azure-ad.sgnl.yaml

```diff
@@ -8,19 +8,16 @@ defaultSyncFrequency: HOURLY
 defaultSyncMinInterval: 1
 defaultApiCallFrequency: SECONDLY
 defaultApiCallMinInterval: 1
-adapter:
-  hostname: "adapter.sgnl.svc.cluster.local"
-  port: 80
-  config: "ewoJInZlcnNpb24iOiAxLAoJImNvbmZpZyI6IHsKICAgICAgICAgICAgICAgICJmaWx0ZXJzIjoge30sCgkJInJlcXVlc3RUaW1lb3V0IjogIjEwcyIsCgkJImF6dXJlYWQiOiB7CiAgICAgICAgICAgICAgICAgICAgImFwaVZlcnNpb24iOiAidjEuMCIKICAgICAgICAgICAgICAgIH0KCX0KfQ=="
-  apiVersion: "V1"
-  disableTls: true
-  auth:
-    - oAuth2ClientCredentials:
-        clientId: "{{Input Required: Client-ID}}"
-        scope: "https://graph.microsoft.com/.default"
-        tokenUrl: "https://login.microsoftonline.com/{{Input Required: Tenant-ID}}/oauth2/v2.0/token"
-        authStyle: InParams
-        clientSecret: "{{Input Required: Client-Secret}}"
+type: "AzureAD-1.0.0"
+adapterConfig: "ewoJInZlcnNpb24iOiAxLAoJImNvbmZpZyI6IHsKICAgICAgICAgICAgICAgICJmaWx0ZXJzIjoge30sCgkJInJlcXVlc3RUaW1lb3V0IjogIjEwcyIsCgkJImF6dXJlYWQiOiB7CiAgICAgICAgICAgICAgICAgICAgImFwaVZlcnNpb24iOiAidjEuMCIKICAgICAgICAgICAgICAgIH0KCX0KfQ=="
```

Comment:
Do you mind adding the following decoded json as a single-line comment as an example of what it looks like? 
```
{ "version": 1, "config": { "filters": {}, "requestTimeout": "10s",  "azuread": { "apiVersion": "v1.0" } } }
```